commit 23b9b1e51fee68fbd31585bd5a8f07959a802a8d
Author: Jelmer Vernooij <jelmer@samba.org>
Date:   Mon Apr 27 23:33:52 2015 +0000

    Add set date to tdb manpages.
    
    This makes builds reproducible.
    
    Signed-off-by: Jelmer Vernooij <jelmer@samba.org>

diff --git a/man/ldb.3.xml b/man/ldb.3.xml
index b93d532..e138c71 100644
--- a/man/ldb.3.xml
+++ b/man/ldb.3.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldb.3">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldb</refentrytitle>
diff --git a/man/ldbadd.1.xml b/man/ldbadd.1.xml
index 3a3ed2c..5d6936b 100644
--- a/man/ldbadd.1.xml
+++ b/man/ldbadd.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbadd.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldbadd</refentrytitle>
diff --git a/man/ldbdel.1.xml b/man/ldbdel.1.xml
index d57cf2a..cbb6714 100644
--- a/man/ldbdel.1.xml
+++ b/man/ldbdel.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbdel.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldbdel</refentrytitle>
diff --git a/man/ldbedit.1.xml b/man/ldbedit.1.xml
index fd98c60..22ee705 100644
--- a/man/ldbedit.1.xml
+++ b/man/ldbedit.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbedit.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
     <refmeta>
 	<refentrytitle>ldbedit</refentrytitle>
diff --git a/man/ldbmodify.1.xml b/man/ldbmodify.1.xml
index 5d6b468..87350fd 100644
--- a/man/ldbmodify.1.xml
+++ b/man/ldbmodify.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbmodify.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldbmodify</refentrytitle>
diff --git a/man/ldbrename.1.xml b/man/ldbrename.1.xml
index 102cee4..7066140 100644
--- a/man/ldbrename.1.xml
+++ b/man/ldbrename.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbrename.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldbrename</refentrytitle>
diff --git a/man/ldbsearch.1.xml b/man/ldbsearch.1.xml
index e693a290..8ff19af 100644
--- a/man/ldbsearch.1.xml
+++ b/man/ldbsearch.1.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
 <!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN" "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
 <refentry id="ldbsearch.1">
+<refentryinfo><date>2015-04-26</date></refentryinfo>
 
 <refmeta>
 	<refentrytitle>ldbsearch</refentrytitle>
